src/rpcmining.cpp

```diff
@@ -127,6 +127,9 @@ UniValue getmininginfo(const UniValue& params, bool fHelp)
     {
         const GRC::AccrualComputer calc = GRC::Tally::GetComputer(*cpid, nTime, pindexBest);
 
+        GRC::Magnitude magnitude = GRC::Quorum::GetMagnitude(mining_id);
+
+        obj.pushKV("magnitude_as_of_last_superblock", magnitude.Floating());
```

Comment:
Something like `last_magnitude` perhaps?

Reply:
I like the more descriptive term because that is exactly what it is...

Reply:
last magnitude is vague... what does "last" mean... as of the last superblock, the last stake, the last scraper convergence, etc.?

Reply:
Good point, perhaps just `last_superblock_magnitude`?

Reply:
I have decided to use current_magnitude. This IS the current magnitude... as it is current until the next superblock is staked and validated.

Reply:
Perhaps only `magnitude`, the `getmininginfo` is assumed to be current is it?

It is maybe easier to understand because it matches the GUI.

Reply:
Either way makes sense to me.